tests/manage/pv_services/pvc_snapshot/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import os
+import urllib.request
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.helpers import helpers
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier1,
+    polarion_id,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@polarion_id("OCS-3946")
+class TestPvcClonedStatusOfSmallSizeFiles(E2ETest):
+    """
+    Tests to verify restored PVC status bound or not if
+    files size are smaller.
+    """
+
+    @bugzilla("2039265")
+    def test_pvc_status_after_clone(
+        self,
+        pvc_factory,
+        pod_factory,
+        teardown_factory,
+        snapshot_factory,
+        snapshot_restore_factory,
+    ):
+        """
+        1. Create 1GiB PVC
+        2. Attach PVC to an application pod
+        3. Download Linux kernel and untar it to the directory where PVC is mounted
+        4. Take a snapshot of the PVC.
+        5. Create a new PVC out of that snapshot.
+        6. Verify PVC state
+
+        Args:
+            pvc_factory: A fixture to create new pvc
+            teardown_factory: A fixture to destroy objects
+            pod_factory:
+            snapshot_factory: A fixture to create new snapshot
+            snapshot_restore_factory: A fixture to restore pvc from snapshot
+        """
+
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.tar.xz"
+        download_path = "tmp"
+        dir_path = os.path.join(os.getcwd(), download_path)
+        file_path = os.path.join(dir_path, "linux-4.4.tar.xz")
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+
```

Comment:
you can use for the pod the `constants.PERF_POD_YAML` this pod contain the linux kernel tgz file in it and a script to extract it and multiple is as much as you want.
you can see implementation at https://github.com/red-hat-storage/ocs-ci/blob/15aa891af26f094d5c26115e9c3db99be7624ce2/tests/e2e/performance/csi_tests/test_pvc_clone_performance.py#L482